src/docs/product/issues/issue-details/performance-issues/consecutive-db-queries/index.mdx

```diff
@@ -25,6 +25,10 @@ Once these spans are found, the following must also hold true:
 
 If Sentry isn't detecting a Consecutive DB issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
```

Comment:
```suggestion
If Sentry isn't detecting a consecutive DB issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
```